### ISSUE TITLE
Tilt fails with "Error could not find Tilt web static files at path: ..."

### ISSUE THREAD

**Issue by joseph-zhong** [conda-forge/tilt-feedstock#2]
Issue: Tilt fails with "Error could not find Tilt web static files at path: ..."

```bash
$ tilt up
Tilt started on http://localhost:10350/
v0.16.0-dev, built 2020-07-28
Read TILT_SYNCLET_IMAGE from environment: <redacted>.com/tilt-synclet
Error: Could not find Tilt web static files at path: /Users/runner/miniforge3/conda-bld/tilt_1595982109383/work/web
```

**Comment by nicks** [comment]
ya! I think that's because you're using the "dev" compile invocation instead of the prod one, so it's trying to find the JS/CSS/HTML files on local disk. lemme see if i can fix this

**Comment by joseph-zhong** [comment]
I see that makes sense, yeah I simply used the `make install` command, thanks for looking into this!

**Comment by joseph-zhong** [comment]
I think this is now resolved thanks to #4, thanks again Nick!

```
(ctrldev) ~/work/tilt [(v0.16.1)]$ which tilt
/Users/josephz/miniconda3/envs/ctrldev/bin/tilt
(ctrldev) ~/work/tilt [(v0.16.1)]$ tilt version
v0.16.1, built 
(ctrldev) ~/work/tilt [(v0.16.1)]$ tilt up
Tilt started on http://localhost:10350/
v0.16.1, built 

(space) to open the browser
(s) to stream logs (--hud=false)
(h) to open terminal HUD (--hud=true)
(ctrl-c) to exit
```
